packages/manager/modules/emailpro/src/dashboard/emailpro.service.js

```diff
@@ -1028,6 +1028,15 @@ export default class EmailPro {
     }).then(([associatedDomainName]) => associatedDomainName);
   }
 
+  retrieveMxPlan(serviceName) {
+    return this.OvhHttp.get(
+      `/email/mxplan/${serviceName || this.$stateParams.productId}`,
+      {
+        rootPath: 'apiv6',
+      },
+    );
+  }
```

Comment:
```suggestion
  retrieveMxPlan(serviceName) {
    return this.$http.get(
      `/email/mxplan/${serviceName || this.$stateParams.productId}`,
    ).then(({data}) => data)
  }
```
And why you don't sure to have "serviceName" ?

Reply:
because the call to retrieveMxPlan is made without arguments and can be made with an argument serviceName

So it is optionnal

Reply:
I dont see the need of : can be made with an argument serviceName. 
May be you can remove the serviceName ?

Reply:
removed :)